models/drone.py

```diff
@@ -2,10 +2,7 @@
 
 import sqlalchemy as sa
```

Comment:
The best approach is not to import ALL sqlalchemy, but to use smth like this:
`from sqlalchemy import Column, ...`
and then use it directly.

(what's the difference: 
`import sqlalchemy as sa and using sa.Column `
vs. 
`from sqlalchemy import Column`